File: javascript/3467.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n'), l;
lines.pop()
while(lines.length) {
    l = lines.shift().trim();
    console.log(l == "xxL" ? "Esse eh o meu lugar" : "Oi, Leonard")
}

File: C/3467.c
#include <stdio.h>
#include <string.h>

int main() {
    
    char l, trash_1, trash_2;
    
    while(scanf("%c%c%c\n", &trash_1, &trash_2, &l) != EOF) {
        printf('L' == l ? "Esse eh o meu lugar\n" : "Oi, Leonard\n");
    }
    
    return 0;
}